Lunar/AppDelegate.swift

```diff
@@ -501,6 +520,23 @@ class AppDelegate: NSObject, NSApplicationDelegate, CLLocationManagerDelegate, N
         brightnessAdapter.setContrastPercent(value: percent)
         log.debug("Setting brightness and contrast to \(percent)%")
     }
+    
+    private func increaseBrightness(_ amount: Int8 = 2) {
+        adjustBrightness(amount)
+    }
+    
+    private func decreaseBrightness(_ amount: Int8 = 2) {
+        adjustBrightness(-amount)
+    }
+    
+    private func adjustBrightness(_ amount: Int8) {
+        let currentBrightness = brightnessAdapter.displays.first?.value.brightness.intValue
```

Comment:
`brightnessAdapter.displays.first?.value.brightness.intValue` does not seem to return the display's current brightness reliably. Any ideas why or which function to use instead?